src/Module/EddBkRestApiModule.php

```diff
@@ -0,0 +1,528 @@
+use Dhii\Iterator\NormalizeIteratorCapableTrait;
+use Dhii\Util\String\StringableInterface as Stringable;
+use IteratorIterator;
+use Psr\Container\ContainerInterface;
+use Psr\EventManager\EventManagerInterface;
+use RebelCode\EddBookings\RestApi\Controller\BookingsController;
+use RebelCode\EddBookings\RestApi\Controller\ClientsController;
+use RebelCode\EddBookings\RestApi\Controller\ServicesController;
+use RebelCode\EddBookings\RestApi\Handlers\Bookings\BookingInfoHandler;
+use RebelCode\EddBookings\RestApi\Handlers\Bookings\CreateBookingHandler;
+use RebelCode\EddBookings\RestApi\Handlers\Bookings\DeleteBookingHandler;
+use RebelCode\EddBookings\RestApi\Handlers\Bookings\QueryBookingsHandler;
+use RebelCode\EddBookings\RestApi\Handlers\Bookings\UpdateBookingHandler;
+use RebelCode\EddBookings\RestApi\Handlers\Clients\ClientInfoHandler;
+use RebelCode\EddBookings\RestApi\Handlers\Clients\CreateClientHandler;
+use RebelCode\EddBookings\RestApi\Handlers\Clients\QueryClientsHandler;
+use RebelCode\EddBookings\RestApi\Transformer\CallbackTransformer;
+use RebelCode\EddBookings\RestApi\Transformer\MapTransformer;
+use RebelCode\EddBookings\RestApi\Transformer\NoOpTransformer;
+use RebelCode\EddBookings\RestApi\Transformer\TransformerIterator;
+use RebelCode\Modular\Module\AbstractBaseModule;
+use Traversable;
+use WP_Post;
+
+/**
+ * The REST API module class.
+ *
+ * @since [*next-version*]
+ */
+class EddBkRestApiModule extends AbstractBaseModule
+{
+    /* @since [*next-version*] */
+    use NormalizeIteratorCapableTrait;
+
+    /**
+     * Constructor.
+     *
+     * @since [*next-version*]
+     *
+     * @param string|Stringable         $key                  The module key.
+     * @param string[]|Stringable[]     $dependencies         The module dependencies.
+     * @param ContainerFactoryInterface $configFactory        The config factory.
+     * @param ContainerFactoryInterface $containerFactory     The container factory.
+     * @param ContainerFactoryInterface $compContainerFactory The composite container factory.
+     * @param EventManagerInterface     $eventManager         The event manager.
+     * @param EventFactoryInterface     $eventFactory         The event factory.
+     */
+    public function __construct(
+        $key,
+        $dependencies,
+        $configFactory,
+        $containerFactory,
+        $compContainerFactory,
+        $eventManager,
+        $eventFactory
+    ) {
+        $this->_initModule($key, $dependencies, $configFactory, $containerFactory, $compContainerFactory);
+        $this->_initModuleEvents($eventManager, $eventFactory);
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @since [*next-version*]
+     *
+     * @throws InternalException If an error occurred while trying to read from the config file.
+     */
+    public function setup()
+    {
+        return $this->_setupContainer(
+            $this->_loadPhpConfigFile(RCMOD_EDDBK_REST_API_CONFIG_FILE),
```

Comment:
This introduces coupling to the module wrapper. Either way, this should be replaced with a call to a path mapping container, when that is ready.

Reply:
> This introduces coupling to the module wrapper.